data/json/items/armor/arms_armor.json

```diff
@@ -878,6 +878,54 @@
       }
     ]
   },
+  {
+    "id": "armguard_bronze",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": { "str": "pair of bronze vambraces", "str_pl": "pairs of bronze vambraces" },
+    "description": "A pair of bronze arm guards with a simple leather lining.",
+    "weight": "2270 g",
+    "volume": "5250 ml",
+    "price": 40000,
+    "price_postapoc": 750,
+    "to_hit": 1,
+    "material": [ "bronze", "leather" ],
+    "symbol": "[",
+    "looks_like": "armguard_lightplate",
+    "color": "yellow",
+    "warmth": 15,
+    "material_thickness": 4,
+    "flags": [ "OUTER", "STURDY" ],
+    "armor": [
+      {
+        "covers": [ "arm_l", "arm_r" ],
+        "encumbrance": 5,
+        "coverage": 95,
+        "cover_melee": 95,
+        "cover_ranged": 90,
+        "cover_vitals": 70,
+        "specifically_covers": [ "arm_lower_r", "arm_lower_l" ]
```

Comment:
greaves also cover the knees if you wanted to keep parity.

Reply:
When I searched up images of vambraces, most didn't have elbow protection, which is why I left it out.
I can update the vambraces to cover the elbows as well if that's wanted.

Reply:
nah if they don't they don't, I wouldn't worry about that.